template/koa-rest-api/src/framework/validation.test.ts

```diff
@@ -42,13 +42,9 @@ describe('validate', () => {
       .send({ ...idDescription, id: null })
       .expect(422)
       .expect(({ text }) =>
-        expect(text).toMatchInlineSnapshot(`
-          "Validation failed:
-          {
-            "id": "Expected string, but was null"
-          }.
-          Object should match { id: string; description: string; }"
-        `),
+        expect(text).toMatchInlineSnapshot(
```

Comment:
Because we're responding with JSON, it would be nicer to snapshot the JSON response. You can destructure `body` instead of `text` to accomplish this.